.github/ISSUE_TEMPLATE/config.yml

```
@@ -9,6 +9,9 @@ contact_links:
   - name: Chat with us on Matrix?
     url: https://matrix.to/#/#valkey:matrix.org
     about: We are on Matrix too!
+  - name: Chat with us on Slack?
+    url: https://valkey-oss-developer.slack.com/archives/C071Q2B3RNC
```

```suggestion
    url: https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2nxs51chx-EB9hu9Qdch3GMfRcztTSkQ
```
I think you have to use this link? If you aren't already on the slack you will just get an error.

---

Thanks!